src/main/java/com/landa44/simplerpg/character/attribute/CharacterAttributes.java

```diff
@@ -0,0 +1,94 @@
+package com.landa44.simplerpg.character.attribute;
+
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import com.landa44.simplerpg.attribute.Attribute;
+import com.landa44.simplerpg.character.Character;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.SequenceGenerator;
+import jakarta.persistence.Table;
+
+/**
+ * A class to store the value of the character's attributes.
+ * Represents the relationship between character and attribute.
+ */
+@Entity
+@Table(name = "character_attributes")
+@JsonSerialize(using = CharacterAttributesSerializer.class)
+public class CharacterAttributes {
+    @Id
+    @SequenceGenerator(
+        name = "character_attributes_sequence",
+        sequenceName = "character_attributes_sequence",
+        allocationSize = 1
+    )
+    @GeneratedValue(
+        strategy = GenerationType.SEQUENCE,
+        generator = "character_attributes_sequence"
+    )
+    private Long id;
+
+    private int value;
+
+    @ManyToOne
+    @JoinColumn(name = "attribute_id")
+    private Attribute attribute;
+
+
+    @ManyToOne
+    @JoinColumn(name = "character_id")
+    private Character character;
```

Comment:
You are not consistent with the field declarations, sometimes you put them all together, or as in this case where you have lines between them, event 2 lines